generator/package.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "symbol-typescript-generator",
+    "version": "0.0.1",
+    "description": "symbol typescript buffer generator",
+    "main": "dist/index.js",
+    "typings": "dist/index.d.ts",
+    "scripts": {
+        "build": "npx shx rm -rf build/ && tsc && ts-node index.ts"
+    },
+    "author": "stevenLiu",
+    "license": "Apache-2.0",
+    "devDependencies": {
+        "@types/chai": "^4.2.21",
+        "@types/fs-extra": "^9.0.13",
+        "@types/mocha": "^9.0.0",
+        "@types/n-readlines": "^1.0.2",
+        "@types/node": "^16.4.6",
+        "@typescript-eslint/eslint-plugin": "^4.28.5",
+        "@typescript-eslint/parser": "^4.28.5",
+        "chai": "^4.3.4",
+        "coveralls": "^3.1.1",
+        "eslint": "^7.32.0",
+        "eslint-config-prettier": "^8.3.0",
+        "eslint-plugin-prettier": "^3.4.0",
+        "eslint-plugin-tsdoc": "^0.2.14",
+        "mocha": "^9.0.3",
+        "nyc": "^15.1.0",
+        "prettier": "^2.3.2",
+        "prettier-plugin-organize-imports": "^2.3.3",
+        "shx": "^0.3.3",
+        "ts-mockito": "^2.6.1",
+        "ts-node": "^10.1.0",
+        "tsconfig-paths": "^3.10.1",
+        "typedoc": "^0.21.4",
+        "typescript": "^4.3.5"
+    },
+    "dependencies": {
+        "bigint-buffer": "^1.1.5",
+        "fs-extra": "^10.0.0",
```

Comment:
Remove `fs-extra`, the generator is not using anything special. You can use standard 'fs' node module.